ide/coqide/protocol/xmlprotocol.ml

```diff
@@ -15,8 +15,8 @@ let protocol_version = "20210506"
 
 (** See xml-protocol.md for a description of the protocol. *)
 
-type msg_format = Richpp of int | Ppcmds
-let msg_format = ref (Richpp 72)
+type msg_format = Richpp of { width : int; depth : int } | Ppcmds
+let msg_format = ref (Richpp { width = 72; depth = max_int })
```

Comment:
This is changing the protocol, you need to bump `protocol_version` above.

Reply:
Umm, I don't see why this changes the protocol, can you be more specific?

Reply:
Oops, sorry, I misread the change. Indeed this is fine as-is.